src/filter-dropdown.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export default class FilterDrop extends React.Component {
```

Comment:
This would look a lot better as a `<select>` element with `<option>`s.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select